### ISSUE TITLE
BINDetect calculating footprinting scores to TFBS and global binding threshold

### ISSUE THREAD

**Issue by mtoninel** [loosolab/TOBIAS#97]
Hello Mette,

Thanks for this amazing tool! I have a couple of questions regarding BINDetect which came up while I was reading the supplementary informations of the TOBIAS paper.

In the first place, I understood the way the footprint score is calculated for each position along the genome (position that is referred to as "i" in the supplementary equations, which I'm guessing is each single base), but I'm struggling to understand how then we end up with a single footprint score for each binding site (i.e. TFBS) which is composed by multiple bases. In other words, how does the 'matching' between the footprint scores and the binding site take place? What is the operation performed across the different single-position scores?

Secondly, I was wondering how is the global bound/unbound threshold calculated?
In other words, when the BINDetect section of the supplementary says:

> "In the second step of the algorithm, every TF binding site found (for each motif given as input) is split into bound and unbound sites based on a score threshold per condition. The threshold is set at the level of significance of a normal-distribution fit to the background distribution of scores (user-defined p-value)."

How can a single background distribution of scores be built starting from the different score distributions of the single conditions if this is really the case?

Thanks,

Mattia

**Comment by mtoninel** [comment]
Hi Mette!
 
Yeah the whole concept seems much clearer now!
Thanks for your helpful reply and thanks again for Tobias! 

Mattia
